src/rules/no-duplicate-definitions.js

```diff
@@ -114,13 +123,24 @@ export default {
 				}
 
 				if (footnoteDefinitions.has(node.identifier)) {
+					const firstDefinition = footnoteDefinitions.get(
```

Comment:
```suggestion
					const firstFootnoteDefinition = footnoteDefinitions.get(
```

Maybe adding a `Footnote` would be clearer here.